### ISSUE TITLE
README problem

### ISSUE THREAD

**Issue by Neustradamus** [OpenDUNE/OpenDUNE#343]
I see in the README:

The OpenDUNE team (in alphabetical order):
  Albert Hofkamp (Alberth)             - Refactoring Dude
  Loic Guilloux (glx)                  - Windows Guru
  Patric Stout (TrueBrain)             - Lead Developer
  Steven Noorbergen (Xaroth)           - Lead Manager
  Thomas Bernard                       - Developer

In the README, I think that we can change "Thomas Bernard" by "miniupnp" because, it is the miniupnp project which contribute in OpenDUNE not a person.

I have already explained here the problem:
- https://github.com/miniupnp/miniupnp/issues/436

Thanks in advance.

**Comment by TrueBrain** [comment]
Last I checked, it are people who write code; not projects.

This is a very weird and odd request. First off all, if such change will ever be made, it will be by request of @miniupnp ; not some random dude on the Internet.

But besides all that, especially the ticket you link, gives no reason what so ever to change credits from Thomas to a project. It even hardens the fact that the current credit-list is correct: he made the contributions, not the project. As he was the one typing .. that you want his username to be changed into something else .. well .. don't drag this project into your politics, I would say ;) But even with a rename of username, the best you can hope for that we finally add a nickname after his name. The information given gives no reason for anything else.

In short, I see no reason to process this request. Either you haven't given all information, or this request is invalid. And again, this is a very odd request ........

**Comment by Neustradamus** [comment]
Hope that @miniupnp will understand the problem that I have signaled several (6) months ago.
- https://github.com/miniupnp/miniupnp/issues/436

The problem can be solved in 5 minutes, it is really easy.

OpenDUNE is good here: https://github.com/OpenDUNE.

**Comment by TrueBrain** [comment]
That "problem" (which really is not a problem), has nothing to do with his contributions here. Your quest here is invalid. So please stop trying ...
